src/fabric-1.19.2/procedures/world_send_chat.java.ftl

```diff
@@ -1,5 +1,5 @@
 if (!world.isClientSide()) {
 	MinecraftServer _mcserv = ServerLifecycleHooks.getCurrentServer();
```

Comment:
Can you also find a fix for the `ServerLifecycleHooks.getCurrentServer()` part? ServerLifecycleHooks doesn't exist anymore, so the block still fails to build.

Reply:
This doesn't seem to be the case as I've used it myself?

Reply:
![image](https://user-images.githubusercontent.com/38427877/206273871-557b1c64-f614-4040-92f2-00dc08e09f1d.png)
I checked the package and there is no hook class with this name.